cdap-ui/app/login/components/Card/index.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+import React, {Component, PropTypes} from 'react';
+var classNames = require('classnames');
+require('./Card.less');
+export default class Card extends Component {
+  constructor(props) {
+    super(props);
+    this.props = props;
+  }
+  render() {
+    var statusClasses = classNames('card-status',
+      { 'error' : this.props.error },
+      { 'success': this.props.success },
+      { 'hide': !this.props.error && !this.props.success }
+    );
+    return (
+      <Card-Wrapper>
```

Comment:
Are we still going to go with this? I think React still trying to look for this class